docs/docs/overview.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Overview
 id: overview
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/overview.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```

Comment:
should be updated to https://docs.secretless.io/Latest/en/Content/Overview/overview.htm